### ISSUE TITLE
`uninlined_format_args` misses multi-line macros

### ISSUE THREAD

**Issue by Nemo157** [rust-lang/rust-clippy#9719]
### Summary

When the macro arguments span multiple lines they are missed by the lint.

### Lint Name

uninlined_format_args

### Reproducer

I tried this code ([playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=94a3a685c73c4b177a7b72e2be5bc6c2)):

```rust
pub fn foo() {
    let a = 1;
    format!("{}",
        a
    );
}
```

I expected to see this happen: an `uninlined_format_args` warning

Instead, this happened: no warning

### Version

```text
clippy: 0.1.66 (2022-10-24 758f196)
```

**Comment by kraktus** [comment]
Hello, this is done on purpose for now https://github.com/rust-lang/rust-clippy/pull/9599
